### DISCUSSION TITLE
Audio PDM data transmission using OpenThread

### DISCUSSION THREAD

**Discussion by nikhil1983** [openthread/openthread#8323] [Q&A] [answered]
Hello I have interfaced microphone (MP34DT05) with nRF52840 SOC using PDM. The audio sampling rate is around 16K. 

So I would like to know if I can do continuous transmission of audio for say 1 minute using thread protocol ?

**Comment by jwhui** [comment, marked as answer]
It largely depends on the data rate and latency requirements. The data rate is largely determined by sampling rate and compression ratio.

While Thread's physical layer operates at 250 kbps data rate, the usable application-layer data rate is typically in the tens of kilobits per second. Of course, that capacity is shared with all devices in the Thread network.

**Comment by nikhil1983** [comment]
So if my sensor device sampling rate is 16KHz i.e 16000 pdm data per
second. So is it still possible to transmit this much data in single second
?

**Reply by jwhui** [reply]
If you are talking about raw 16-bit samples, 16000 * 16 = 256,000 bits per second - see my comment in https://github.com/openthread/openthread/discussions/8323#discussioncomment-3948999.
